lib/presentation/mixins/handle_registration_mixin.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:fluffychat/config/app_config.dart';
+import 'package:fluffychat/widgets/twake_app.dart';
+import 'package:flutter_web_auth_2/flutter_web_auth_2.dart';
+import 'package:matrix/matrix.dart';
+
+mixin HandleRegistrationMixin {
+  void handleRegistrationDone(String uri) async {
+    Logs().i('handleRegistrationDone: uri: $uri');
+    TwakeApp.router.go('/home', extra: uri);
+  }
+
+  void register(String url) async {
+    final uri = await FlutterWebAuth2.authenticate(
```

Comment:
in Android, dialog show or not?

Reply:
ahh, it use only for iOS, please more specific in method name.

Reply:
okay